docs/tre-admins/environment-variables.md

```diff
@@ -42,7 +44,7 @@
 | `FIREWALL_SKU` | Optional. The SKU of the Azure Firewall instance. Default value is `Standard`. Allowed values [`Basic`, `Standard`, `Premium`]. See [Azure Firewall SKU feature comparison](https://learn.microsoft.com/en-us/azure/firewall/choose-firewall-sku). |
 | `APP_GATEWAY_SKU` | Optional. The SKU of the Application Gateway. Default value is `Standard_v2`. Allowed values [`Standard_v2`, `WAF_v2`] |
 | `CUSTOM_DOMAIN` | Optional. Custom domain name to access the Azure TRE portal. See [Custom domain name](custom-domain.md). |
-
+| `ENABLE_CMK_ENCRYPTION` | If set to `true`, customer-managed key encryption will be enabled for all supported resources. |
```

Comment:
@yuvalyaron is there additional configuration required - such as KeyVault etc? Would be good to have something in the docs detailing how we actually make this "work". Thanks.

Reply:
These are also env vars?

```hcl
variable "enable_cmk_encryption" {
  type        = bool
  description = "A boolean indicating if customer managed keys will be used for encryption of supporting resources"
  default     = false
}

variable "external_key_store_id" {
  type        = string
  description = "ID of external Key Vault to store CMKs in (only required if enable_cmk_encryption is true)"
  default     = null
}

variable "encryption_kv_name" {
  type        = string
  description = "Name of Key Vault for encryption keys, required only if external_key_store_id is not set (only used if enable_cmk_encryption is true)"
  default     = null
}

variable "kv_encryption_key_name" {
  type        = string
  description = "Name of Key Vault Encryption Key (only used if enable_cmk_encryption is true)"
  default     = "tre-encryption"
}
```

Reply:
- I'll add a section in the docs.
- The env vars you mentioned are documented here

Reply:
added a section in the docs